doc/api/deprecations.md

```diff
@@ -1274,14 +1335,28 @@ changes:
     description: Documentation-only deprecation.
 -->
 
-Type: Documentation-only
+Type: Runtime
+
+The [`util.log()`][] API has been deprecated. Instead,
```

Comment:
The `util.log` is basically just console.log + a timestamp, I think the alternative can simply be:

```js
// Or do more sophisticated formatting of the time with a third-party library
util.log(new Date().toLocaleString(), message);
```